aikau/src/main/resources/alfresco/layout/Twister.js

```diff
@@ -65,6 +111,24 @@ define(["dojo/_base/declare",
       templateString: template,
 
       /**
+       * The CSS class applied when the twister is opened (and removed when closed).
+       *
+       * @instance
+       * @type {string}
+       * @default "alfresco-layout-Twister-open"
+       */
+      CLASS_OPEN: "alfresco-layout-Twister-open",
+      
+      /**
+       * The CSS class applied when the twister is closed (and removed when opened).
+       *
+       * @instance
+       * @type {string}
+       * @default "alfresco-layout-Twister-closed"
+       */
+      CLASS_CLOSED: "alfresco-layout-Twister-closed",
```

Comment:
Why do we need a closed class when it can only have two states? Surely the absence of open is closed (or vice versa, if preferred)?